spirv_cross/src/msl.rs

```diff
@@ -65,21 +65,23 @@ pub enum Version {
     V1_1,
     V1_2,
     V2_0,
+    V2_1,
 }
 
 impl Version {
     fn as_raw(&self) -> u32 {
         use self::Version::*;
         match *self {
             V1_0 => 10000,
-            V1_1 => 10100,
-            V1_2 => 10200,
+            V1_1 => 10001,
```

Comment:
Is this really correct for minor versions? I based it on https://github.com/KhronosGroup/SPIRV-Cross/blob/162eee632599cd077972ee1d88b341eedbcfdf27/spirv_msl.hpp#L181

Reply:
Metal documentation disagrees: https://developer.apple.com/documentation/metal/mtllanguageversion/mtllanguageversion1_2

Reply:
But this `as_raw` is used to pass the version `uint` to SPIRV-Cross, not necessarily needing to match the actual Metal value

Reply:
wow, that's unexpected. I'll fix this back